lib/components/button/button.a11y.test.ts

```diff
@@ -14,8 +14,8 @@ describe("button", () => {
             /s-btn-(light|dark).*?badge/,
             // matches tests with a badge in highcontrast-light modes, excluding filled, danger, github, facebook, sm, or xs
             /s-btn-highcontrast-light-(?!.*(filled|danger|github|facebook|sm|xs)).*?badge/,
-            // matches tests with a badge in highcontrast-light modes, are muted and/or outlined, and are sm or xs
-            /s-btn-highcontrast-light-(?:muted-outlined-|muted-|outlined-)?(?:sm|xs).*?badge/,
+            // matches tests with a badge in highcontrast-light modes, are muted or featured and/or outlined, and are sm or xs
```

Comment:
Out of curiosity what's the contrast ratio for the failing tests? 
By glancing at the UI it doesn't seem we are a lot off from the 7:1 target.

Reply:
Yeah, the featured variant seems to land at 6.61 in the failing cases. It's from the button badge opacity being set to `.8` in high contrast mode.

_Personally_, I'd be fine to bump that to `.85` or `1`, but we'd either need to add an exception for the `featured` variant for this or change the button badge opacity for all buttons in HC mode. I'd prefer the latter but I'd want to get approval from design before making either change. Thoughts?

Reply:
I also think the latter is the better option. Then let's consider this out of scope for this PR and maybe let's have a chat with design about it at some point. Thanks